subsys/net/openthread/Kconfig.defconfig

```diff
@@ -152,7 +155,8 @@ endif # !(OPENTHREAD_BORDER_AGENT || OPENTHREAD_COMMISSIONER || OPENTHREAD_COAPS
 
 config MBEDTLS_HEAP_SIZE
 	int
-	default 12440
+	default 15360 if OPENTHREAD_NRF_SECURITY_PSA
```

Comment:
Note here, that is an experimental value. I didn't do any reliable measurements, but for 12440 it didn't work for me.  We can do some measurements using watermarks, or at least add a note here that the number may be possibly reduced in the future.

Reply:
added a note. We'll work on that as next step